controllers/configurationpolicy_controller.go

```diff
@@ -2792,8 +2793,12 @@ func handleDiff(
 	case policyv1.RecordDiffInStatus:
 		return computedDiff
 	case policyv1.RecordDiffCensored:
-		return `# This diff may contain sensitive data. The "recordDiff" field must be set to "InStatus" ` +
-			`to record a diff.`
+		return fmt.Sprintf(
+			`# This diff may contain sensitive data. The spec["object-templates"][%d]["recordDiff"] field must be set `+
```

Comment:
I have mixed feeling about being this specific. I think `spec.object-templates[].recordDiff` is a bit easier to look at in my opinion, and I'm not sure how much value there is in providing the index, especially with `object-templates-raw` in the mix.
```suggestion
			`# This diff may contain sensitive data. The spec["object-templates"][%d]["recordDiff"] field must be set `+
```